### PR TITLE
Acm/dscs-121 Add additional properties to sections

### PR DESCRIPTION
**Pull request by AaronCMuller**
@terenceingram @bsingh-nla Please take a look.

### CODE REVIEW

File: src/amberdb/model/Section.java [hunk 1]
```diff
@@ -44,6 +44,36 @@
 	
 	@JavaHandler
 	public int countExistsOns();
+
+	@Property("abstract")
+	public String getAbstract();
+
+	@Property("abstract")
+	public void setAbstract(String abstractText);
```
Comment (shuangzhou) [inline src/amberdb/model/Section.java]: @AaronCMuller , this field already exist in Work, Section automatically inherit this.

File: src/amberdb/model/Section.java [hunk 2]
```diff
@@ -44,6 +44,36 @@
...
+
+	@Property("captions")
+	public List<String> getCaptions();
+
+	@Property("captions")
+	public void setCaptions(List<String> captions);
+
+	@Property("advertising")
+	public boolean isAdvertising();
```
Comment (shuangzhou) [inline src/amberdb/model/Section.java]: @AaronCMuller could you please change return type to Boolean?

File: src/amberdb/model/Section.java [hunk 3]
```diff
@@ -44,6 +44,36 @@
...
+
+	@Property("advertising")
+	public void setAdvertising(boolean advertising);
+
+	@Property("illustrated")
+	public boolean isIllustrated();
```
Comment (shuangzhou) [inline src/amberdb/model/Section.java]: @AaronCMuller could you please change return type to Boolean?

Comment (AaronCMuller) [inline src/amberdb/model/Section.java]: @shuangzhou Can you tell me what difference it makes?

Comment (shuangzhou) [inline src/amberdb/model/Section.java]: @AaronCMuller the field might not be always set, from memory, this will return null instead of a boolean value.

**Pull request merged**